docs/guides/creatingadls.md

```diff
@@ -83,52 +139,137 @@ spec:
+        name: user-container
 ```
 
-## Viewing the Results of a Dead Letter Sink
+Apply these changes to the cluster.
+
+
+### Step 7: Test the Bridge
 
-Now that we have all the parts in place, we can monitor the events that are being sent to the dead letter sink. We can do this in two ways:
+Now with everything in place if we retrieve the current pods in the namespace of deployment, we should see our `event-display` and `event-failure-capture` services running.
+
+
+```cmd
+$ kubectl get pods
+NAME                                              READY   STATUS    RESTARTS   AGE
+event-display-00001-deployment-689f6d648d-tx4r5   2/2     Running   0          34s
+sockeye-00001-deployment-6f488dcd5b-w8zmb         2/2     Running   0          34s
+```
 
-1. View the pod logs of the `sockeye` service:
-  * `kubectl get pods` will show the pods that are running. Retrieve the sockeye pod name from the output.
-  * `kubectl logs <SOCKEYE_POD_NAME> user-container` By replacing the `<SOCKEYE_POD_NAME>` with the pod name you can view the logs of the sockeye pod.
+ After 60 secconds we should see the `event-failure-capture` service spin down, leaving us with only the `event-display` service running.
 
-2. View the web service exposed by the `sockeye` service:
-  * `kubectl get ksvc` will show the KSVC's that are running. Retrieve the sockeye public URL from the `URL` column and navigate to it in your browser.
+```cmd
+$ kubectl get pods
+NAME                                              READY   STATUS    RESTARTS   AGE
+event-display-00001-deployment-689f6d648d-tx4r5   2/2     Running   0          2m
+```
+
+View the logs of the `event-display` service, to verify that the events are being recieved from the [PingSource](https://knative.dev/docs/eventing/sources/ping-source/) object.
+
+```cmd
+$ kubectl logs event-display-00001-deployment-689f6d648d-tx4r5 user-container
+
+2022/05/31 20:45:00 Broadasting to 0 clients: {"data":{"name":"triggermesh"},"id":"0e429ad9-cf9d-451d-98ce-886fa92acc35","knativearrivaltime":"2022-05-31T20:45:00.250764419Z","source":"/apis/v1/namespaces/delme/pingsources/ping-sockeye","specversion":"1.0","time":"2022-05-31T20:45:00.250270401Z","type":"dev.knative.sources.ping"}
+got Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.sources.ping
+  source: /apis/v1/namespaces/delme/pingsources/ping-sockeye
+  id: 5c448228-f413-4cbd-b9a4-1c1472c1b96a
+  time: 2022-05-31T20:46:00.135993596Z
+Extensions,
+  knativearrivaltime: 2022-05-31T20:46:00.136447484Z
+Data,
+  {"name": "triggermesh"}
+
+```
+
+Here we can see that the events are being delivered and the [Bridge](https://docs.triggermesh.io/concepts/) is functioning properly.
+
+
+### Step 8: Break the Bridge
+
+Break the [Bridge](https://docs.triggermesh.io/concepts/) by removing the `event-display` [Service](https://github.com/knative/specs/blob/main/specs/serving/knative-api-specification-1.0.md#service).
+
```

Comment:
This does not "break" the bridge. You should delete the service to break the flow.

Reply:
thats what i did..